libs/common/src/response-messages/index.ts

```diff
@@ -68,7 +68,8 @@ export const ResponseMessages = {
             userNotFound: 'User not found for the given organization',
             updateUserRoles: 'Unable to update user roles',
             deleteOrg: 'Organization not found',
-            notFound: 'Organization agent not found'
+            notFound: 'Organization agent not found',
```

Comment:
Replace "notFound" with more meaningful name.

Reply:
As discussed with @vivekayanworks. We will resolved this in subsequent PR